ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/TableController.java

```diff
@@ -19,8 +19,10 @@
 package com.netease.arctic.server.dashboard.controller;
 
 import com.google.common.util.concurrent.ThreadFactoryBuilder;
```

Comment:
There is a guava dependency,  please fix it with iceberg guava shade instead.

Reply:
done